sqlparser/ast_test.go

```diff
@@ -535,6 +536,35 @@ func TestTableIdentMarshal(t *testing.T) {
 	}
 }
 
+func TestTableIdentValueForConfig(t *testing.T) {
+	str := TableIdent{
+		quote: 34,
+		v:     "table",
+	}
+	b, err := json.Marshal(str)
```

Comment:
why do we use json package instead of sqlparser.String?

Reply:
Just copied from another test.